android/datastore.md

```diff
@@ -10,23 +10,22 @@ title: DataStore
 <br />
 
 **Note**
-Amplify Android is in preview mode and not intended for production usage at this time. We welcome feedback to improve your experience in using Amplify Android.
+Amplify Android is in preview mode and is not intended for production usage at this time. We welcome feedback to improve your experience in using Amplify Android.
 {: .callout .callout--warning}
 
 # DataStore
 
-Amplify DataStore provides a persistent on-device storage repository for you to write, read, and observe changes to data if you are online or offline, and seamlessly sync to the cloud as well as across devices. Data modeling for your application is using GraphQL and converted to **Models** that are used in JavaScript, iOS, or Android applications. You can use DataStore for your offline use cases in a "local only" mode without an AWS account or provision an entire backend using AWS AppSync and Amazon DynamoDB. DataStore includes *Delta Sync* using your GraphQL backend and several conflict resolution strategies.
-
+Amplify DataStore provides a persistent on-device storage repository for you to write, read, and observe changes to your application data. DataStore works both online and offline. When online, DataStore seamlessly synchronizes data with the cloud, and across devices. You model your application data by defining types in the GraphQL schema definition language (SDL). This schema is automatically converted to Java-language **Model** classes that you can code into your business logic. You can use DataStore for offline use cases in a "local only" mode. In local-only mode, you do not need an AWS account, nor do you need to provision any AppSync or DynamoDB resources. DataStore enables lightweight synchronization and flexible conflict resolution strategies when paired with AWS AppSync.
```

Comment:
Any reason why you pulled Delta Sync out? This was here to raise awareness to customers up front of the feature rather than later.

Reply:
Makes sense! I'll add it back in.